src/client/features/search/index.js

```diff
@@ -30,7 +30,7 @@ class Search extends React.Component {
       searchResults: [],
       loading: false,
       showFilters: false,
-      landingShowMore: false,
+      landingShowMore: [false,false],
```

Comment:
what is the purpose for using two values here?

Reply:
One for each of the show more options.